src/Umbraco.PublishedCache.NuCache/Property.cs

```diff
@@ -382,9 +361,9 @@ private class CacheValue
 
     private class CacheValues : CacheValue
     {
-        private Dictionary<CompositeStringStringKey, CacheValue>? _values;
+        private readonly object _locko = new();
```

Comment:
The "For" method is always called for CacheValues, so `_values` can be created immediately. Then it's possible avoid this `locko`.

Reply:
> `For(string? culture, string? segment)` can be called multiple times on the same `CacheValues` instance for different culture/segment permutations. `_values` will only be null the first time.

Yes, but it doesn't matter. .For is **always** called, so _values is always initialized. There's no point in adding a lock, just create it automatically. I'll give a suggestion.